lib/taskana-core/src/main/java/pro/taskana/task/api/models/Task.java

```diff
@@ -65,100 +67,106 @@
+   * @param customAttributes a Map that contains the custom attributes
    */
   void setCustomAttributeMap(Map<String, String> customAttributes);
 
   /**
-   * Returns a Map of Callback info.
+   * Returns the callbackInfo.
    *
-   * @return callbackInfo as {@link Map}
+   * @return callbackInfo as Map
    */
   Map<String, String> getCallbackInfo();
 
   /**
-   * Sets a Map of callback info.
+   * Sets the callbackInfo.
    *
-   * @param callbackInfo a {@link Map} that contains the callback info
+   * @param callbackInfo a {@link Map} that contains the callback information
    */
   void setCallbackInfo(Map<String, String> callbackInfo);
 
   /**
-   * Sets the value for custom Attribute.
+   * Sets the value for the specified {@linkplain TaskCustomField customField}.
    *
-   * @param customField identifies which custom attribute is to be set.
-   * @param value the value of the custom attribute to be set
+   * @param customField identifies which {@linkplain TaskCustomField customField} is to be set
+   * @param value the value of the {@linkplain TaskCustomField customField} to be set
    * @deprecated Use {@link #setCustomField(TaskCustomField, String)} instead
    */
   void setCustomAttribute(TaskCustomField customField, String value);
 
   /**
-   * Sets the value for custom field.
+   * Sets the value for the specified {@linkplain TaskCustomField customField}.
    *
-   * @param customField identifies which custom field is to be set.
-   * @param value the value of the custom field to be set
+   * @param customField identifies which {@linkplain TaskCustomField customField} is to be set.
+   * @param value the value of the {@linkplain TaskCustomField customField} to be set
    */
   void setCustomField(TaskCustomField customField, String value);
 
   /**
-   * Add an attachment.<br>
-   * NULL will be ignored and an attachment with the same ID will be replaced by the newer one.<br>
+   * Add an {@linkplain Attachment}.<br>
+   * NULL will be ignored and an {@linkplain Attachment} with the same id will be replaced by the
+   * newer one.<br>
    *
    * @param attachment the {@link Attachment attachment} to be added to the Task
    */
   void addAttachment(Attachment attachment);
 
   /**
-   * Return the attachments for this Task. <br>
-   * Do not use List.add()/addAll() for adding Elements, because it can cause redundant data. Use
+   * Return the {@link Attachment attachment} for the Task. <br>
+   * Do not use List.add()/addAll() for adding elements, because it can cause redundant data. Use
    * addAttachment(). Clear() and remove() can be used, because it's a controllable change.
    *
-   * @return the {@link List list} of {@link Attachment attachments} for this Task
+   * @return the List of {@link Attachment attachments} for this Task
    */
   List<Attachment> getAttachments();
 
   /**
-   * Sets the external business process id.
+   * Sets the external businessProcessId.
```

Comment:
Is this id external? I would change it to "associated".

Reply:
Ok